### PR TITLE
v1.42.0

### PR DESCRIPTION
**Pull request by ELundby45**
# grpcio-tools v1.42.0

- Update to match current grpio-feedstock version on defaults

## Changes
- Update version and SHA
- Reset build number
- Add setuptools + wheel dependencies
- Add pip check

## Notes
- This is needed for python 3.11 buildout

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,24 +26,30 @@ requirements:
   host:
     - grpcio =={{ version }}
     - pip
-    - protobuf >=3.6.0
+    - setuptools
+    - wheel
+    - protobuf
```
Comment (cbouss) [inline recipe/meta.yaml:31]: libprotobuf is pinned in the cbc, but not protobuf. In that case I would still add the corresponding pinning.

Comment (ELundby45) [reply recipe/meta.yaml]: Ah! Good catch. Thanks!

Comment (cbouss) [reply recipe/meta.yaml]: I think it should be 3.20.3 (as that's what is pinned on cbc and being built for 3.11)

Comment (ELundby45) [reply recipe/meta.yaml]: Thanks. I'm having a day 😵

**Review by cbouss — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**